apiary.apib

```diff
@@ -3047,6 +3050,33 @@ owner or have edit permissions.
 
 + Response 204
 
+## Add default subject link [/collections/{id}/links/default_subject]
+Links a default subject to a collection. This subject's first media
+location URL will be included in the serialized collection and used
+as the thumbnail. Update this attribute with `null` to use the first
```

Comment:
Should the backend be taking care of that default? It doesn't seem to be in this PR.